### PR TITLE
[term entry]: d3 styling .attr()

### LINKED ISSUE #3961: [Term Entry] D3 Styling .attr()
### Reason/inspiration (optional)

We would like a new term entry on the `.attr()` method for [D3](https://www.codecademy.com/resources/docs/d3): Styling. The entry should go in a new file under `docs/content/d3/concepts/styling/terms/attr/attr.md`.

The entry should describe what the method does. Be sure to include the following sections:

- A "Syntax" section with a ` ```pseudo` code snippet.
- An "Example" section with a ` ```js `code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by vrun1208**
### Description

closes #3961 
Added new term entry - d3 styling .attr()

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/d3/concepts/styling/terms/attr/attr.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+The **`.attr()`** method is used to set or get attributes of selected elements. This method is commonly used for styling elements by setting their attributes.
+
+## Syntax
+
+```pseudo
+selection.attr("attributeName", "attributeValue");
+```
+
+The `.attr()` function takes two parameters, first is the name of attribute and second is the corresponding value or a callback function.
```
Comment (avdhoottt) [inline content/d3/concepts/styling/terms/attr/attr.md:23]: Explain the parameters separately using a bullet list.

File: content/d3/concepts/styling/terms/attr/attr.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+## Example
+
```
Comment (avdhoottt) [inline content/d3/concepts/styling/terms/attr/attr.md:26]: Add a line describing the example.

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@vrun1208 Hey, Some changes from my side!! I might suggest some more changes later.

**Comment by vrun1208** [comment]
@avdhoottt , I made the changes as you suggested

File: content/d3/concepts/styling/terms/attr/attr.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+In the example given below, the value `5` is set to the attribute `radius` which turns the radius of `circle` to 5 units.
+
+```js
+d3.select('circle').attr('radius', 5);
+```
+
+The example given below demonstrates the arguement as a callback function. It is used to dynamically compute attribute value based on the data given. Here function is used to calculate parameter of circle. The callback function is passed the current data and returns the attribute value.
```
Comment (avdhoottt) [inline content/d3/concepts/styling/terms/attr/attr.md:36]:
```suggestion
The example given below demonstrates the argument as a callback function. It is used to dynamically compute attribute value based on the data given. Here the function is used to calculate the parameter of the circle. The callback function is passed the current data and returns the attribute value.
```

File: content/d3/concepts/styling/terms/attr/attr.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+## Example
+
+In the example given below, the value `5` is set to the attribute `radius` which turns the radius of `circle` to 5 units.
+
+```js
+d3.select('circle').attr('radius', 5);
+```
+
+The example given below demonstrates the argument as a callback function. It is used to dynamically compute attribute value based on the data given. Here the function is used to calculate the parameter of the circle. The callback function is passed the current data and returns the attribute value.
```
Comment (avdhoottt) [inline content/d3/concepts/styling/terms/attr/attr.md:36]:
```suggestion
The following example shows how to use the argument as a callback function. Based on the provided data, it is used to compute attribute values dynamically. Here, the circle's parameter is calculated using the function. 
```

**Review by avdhoottt — APPROVED** [review]
Ready for the second review.

File: content/d3/concepts/styling/terms/attr/attr.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+- `attributeValue`: Value of the attribute or a callback function.
+
+## Example
+
+In the example given below, the value `5` is set to the attribute `radius` which turns the radius of `circle` to 5 units.
+
+```js
+d3.select('circle').attr('radius', 5);
+```
+
```
[the remark this answers is not in the document]
Comment (vrun1208) [reply content/d3/concepts/styling/terms/attr/attr.md:35]: Can we use image in output as there is no written output for that

[the remark this answers is not in the document]
Comment (dakshdeepHERE) [reply content/d3/concepts/styling/terms/attr/attr.md:35]: You sure can just remember to put the gif or png in the media folder

[the remark this answers is not in the document]
Comment (dakshdeepHERE) [reply content/d3/concepts/styling/terms/attr/attr.md:35]: @vrun1208 can I get any update on the media addition?

[the remark this answers is not in the document]
Comment (vrun1208) [reply content/d3/concepts/styling/terms/attr/attr.md:35]: @dakshdeepHERE media added 👍, check out the latest commit

**Review by dakshdeepHERE — APPROVED** [review]
LGTM 🚀 Thanks for the commits @vrun1208 :)

**Pull request merged**